docs/core/diagnostics/runtime-exception-events.md

```diff
@@ -0,0 +1,152 @@
+# Exception Events
+These runtime events capture information about exceptions that are thrown.
+
+This category consists of the following events:
+
+- [ExceptionThrown_V1 Event](#exceptionthrown_v1-event)
+- [ExceptionCatchStart Event](#exceptioncatchstart-event)
+- [ExceptionCatchStop Event](#exceptioncatchstop-event)
+- [ExceptionFinallyStart Event](#exceptionfinallystart-event)
+- [ExceptionFinallyStop Event](#exceptionfinallystop-event)
+- [ExceptionFilterStart Event](#exceptionfilterstart-event)
+- [ExceptionFilterStop Event](#exceptionfilterstop-event)
+- [ExceptionThrownStop Event](#exceptionthrownstop-event)
```

Comment:
This is just a repeat of the H2 headers, these are automatically provided on the right-side of the page.

```suggestion
```